start.py

```diff
@@ -40,17 +41,15 @@ def download_jenkins():
                 raise
         os.rename(tmp_file, JENKINS_WAR)
 
+
 if __name__ == "__main__":
     download_jenkins()
 
     try:
-        # for more info see:
-        # http://www.virtualenv.org/en/latest/#using-virtualenv-without-bin-python
         execfile(JENKINS_ENV, dict(__file__=JENKINS_ENV))
         print "Virtual environment activated successfully."
-    except IOError:
-        print "Could not activate virtual environment."
-        print "Exiting."
+    except Exception as ex:
+        print 'Could not activate virtual environment at "%s": %s.' % (JENKINS_ENV, str(ex))
         sys.exit(IOError)
```

Comment:
That existed already before. So we could re-throw the exception or still call `exit()`. For the latter we could e.g. exit with 1.